h/static/scripts/util/search-text-parser.js

```diff
@@ -106,39 +122,7 @@ function getLozengeValuesAndIncompleteSearchTerms(queryString) {
   };
 }
 
-/**
- * Function which returns individual lozenge from a string.
- *
- * @param {String} queryString A string of query terms.
```

Comment:
Good to know. I may send a follow-up PR that fixes them all. I don't suppose this is lintable?